feature-matrix.json

```diff
@@ -1,11 +1,12 @@
 {
   "Question types (basic functionality)": {
     "text": "✅",
-    "integer": "",
-    "decimal": "",
+    "integer": "✅",
+    "decimal": "✅",
     "note": "✅",
     "select_one": "✅",
     "select_multiple": "✅",
+    "select_*_from_file": "✅",
```

Comment:
Since it appears we missed this before, I'm concerned that we've missed others. Not the biggest deal if we're updating it as we go, but definitely doesn't make me feel very confident in using this as a basis for programmatic form feature support detection.

Reply:
I think it was put as part of `secondary instance (external choice file)` in another section. I went ahead and removed that in a new commit since I think this will be more useful for users.

The XLSForm template is intended to be authoritative but it doesn't include everything that's here because it's not all relevant to XLSForm authoring. I find this format and content pretty useful but let's definitely discuss other ideas you have!

Reply:
I think using the XLSForm template might be a great starting point for programmatic concerns, and would probably help narrow down cross referencing with this as a companion source of truth.